packages/run-protocol/src/makeTracer.js

```diff
@@ -17,8 +17,12 @@ const makeTracer = (name, enable = true) => {
       return infoTick;
     }
     default: {
-      const debugTick = (...args) => {
-        console.log(key, (debugCount += 1), ...args);
+      const debugTick = (optLog, ...args) => {
+        if (optLog.log) {
```

Comment:
I hope we find time after MN-1 to rationalize logging. I don't think we have a ticket yet for that per se. https://github.com/Agoric/agoric-sdk/issues/1318 is to do research.